js/front/apidoc.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Display a SwaggerUI documentation
+ */
+import 'front/bootstrap';
+
+import 'less/udata/swagger.less';
+
+import Vue from 'vue';
+
+import $ from 'expose?$!expose?jQuery!jquery';
```

Comment:
SwaggerUI requires jQuery to be a global. This syntax is from the webpack expose plugin (expose both `$` and `jQuery`)